src/gsi/gsimod.F90

```diff
@@ -1368,23 +1368,17 @@ module gsimod
 !     l_timloc_opt - if true, then turn on time-dependent localization
 !     ngvarloc - number of variable-dependent localization lengths
 !     naensloc - total number of spatial localization lengths and scale separation lengths (should be naensgrp+nsclgrp-1)
-!     i_ensloccov4tim - flag of cross-temporal localization
-!                         =0: cross-temporal covariance is retained
-!                         =1: cross-temporal covariance is zero
-!     i_ensloccov4var - flag of cross-variable localization
-!                         =0: cross-variable covariance is retained
-!                         =1: cross-variable covariance is zero
-!     i_ensloccov4scl - flag of cross-scale localization
-!                         =0: cross-scale covariance is retained
-!                         =1: cross-scale covariance is zero
+!     r_ensloccov4tim - factor multiplying to cross-temporal localization
```

Comment:
Also, those documentation should be kept with any updates if needed.  Here in gsimod.f90 is a good place to give explanation/documentation to those namelist parameters.